master/buildbot/steps/source/git.py

```diff
@@ -111,6 +112,7 @@ def __init__(
         self.branch = branch
         self.method = method
         self.repourl = repourl
+        self.port = port
```

Comment:
This probably needs to be added to `GitStepMixin` as evidenced by the tests that are failing.